public/src/string_class.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Prototype object
+ */
+const StringProtoype = {
+
+  /**
+   * Checks if string contains vowels.
+   * @return {Boolean} Returns true or flase.
+   */
+  hasVowels() {
+    return /[aeiou]/ig.test(this);
```

Comment:
Is it necessary to use `g`, since once there is a vowel then the method should return `true`